opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -1,3 +1,21 @@
+/*
+  Copyright (c) 2014 IRIS AS
```

Comment:
It is good that you fix this file's lack of copyright and license statement. However, I have a few requests:
 - The cpp file also lacks this, I would appreciate it if you would put it in.
 - Both files contains parts that were copied from other files, originally written by me. I therefore request that you also add a "2013 SINTEF" copyright line.
 - You probably want your IRIS copyright to be 2015.